service-model-generator/modules/service-model-generator-ls-extension/src/main/java/io/ballerina/servicemodelgenerator/extension/model/HttpResponse.java

```diff
@@ -19,44 +19,61 @@
 package io.ballerina.servicemodelgenerator.extension.model;
 
 public class HttpResponse {
-    private Value statusCode;
-    private Value body;
-    private Value name;
-    private Value createStatusCodeResponse;
-    private Value type;
+    private Value statusCode = null;
+    private Value body = null;
+    private Value name = null;
+    private Value type = null;
+    private Value headers = null;
```

Comment:
explicitly initializing object fields to null is not necessary because instance variables in a class are automatically set to null by default if they are object references
```suggestion
    private Value statusCode;
    private Value body;
    private Value name;
    private Value type;
    private Value headers;
```